fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1505,7 +1505,30 @@ public class Config extends ConfigBase {
      * In order to avoid unnecessary CLONE, we increase the timeout as much as possible
      */
     @ConfField(mutable = true)
-    public static int quorom_publish_wait_time_ms = 5000;
+    public static int quorom_publish_wait_time_ms = 500;
+
+    /**
+     * FE journal queue size
+     * Write log will fail if queue is full
+     **/
+    @ConfField(mutable = true)
+    public static int journal_queue_size = 1000;
+
+    /**
+     * The maxium size(key+value) of journal entity to write as a batch
+     * Increase this configuration if journal queue is always full
+     * TODO: set default value
+     **/
+    @ConfField(mutable = true)
+    public static int batch_journal_size_mb = 100;
+
+    /**
+     * The maxium number of journal entity to write as a batch
+     * Increase this configuration if journal queue is always full
+     * TODO: set default value
+     **/
+    @ConfField(mutable = true)
+    public static int batch_journal_cnt = 100;
```

Comment:
nit: max_bach_journal_cnt

Reply:
why restrict the batch count?

Reply:
We can make our configure name the scope first. like `metadata.journal.max_batch_cnt`. 
Then if we want to find related configuration, we can get it directly.

Reply:
Done.